src/Controller/ModulesController.php

```diff
@@ -285,4 +285,40 @@ public function relationships($id, $relation)
         $this->set(compact('meta'));
         $this->set(compact('links'));
     }
+
+    /**
+     * Upload a file and store it in a media stream
+     *
+     * @return void
+     */
+    public function upload()
```

Comment:
What about a simple test case for `ModulesController` ??
If it takes too much time we can skip this...

Reply:
skip for now... more tests in separate task, I suggest